src/@next/Tabs/TabStyle.ts

```diff
@@ -98,12 +98,14 @@ export const StyledTabButton = styled.button`
   width: 100%;
   padding: 0 ${space4};
 
-  &:hover {
-    color: ${Neutral.B18};
-  }
+  &[data-disabled='false'] {
```

Comment:
do we have some style overrides for disabled state?

Reply:
Based on the Figma here

![img_v3_02c6_67c26822-2c0a-4573-ba9d-f8ae6d43c1hu](https://github.com/glints-dev/glints-aries/assets/52077891/6b32fc6e-9210-4ae6-8192-610b3c0c7e84)

We don't show the HOVER ui when the tab is disabled, no underline no bolder text

For the tooltip part I will confirm with bernard tomorrow if it's okay to show the tooltip only if the cursor hover on the text (in this case I can just wrap the Tooltip on my Employer project tab content component)

Cause, if I need to show the tooltip as soon as the cursor enter the tab area (even outside the text) I will need to update the Aries again to support it.

Reply:
https://www.loom.com/share/f4ea888bd9924770a071ab8e517d635d

Reply:
what about the cursor?

Reply:
not so sure, will wait for bernard answer tomorrow

Reply:
I scheduled the message lol, don't want to disturb him late at night

Reply:
Updated

- Added custom wrapper, this will allow us to wrap the individual tab with any component, including the tooltip
- Cursor will be default if tab is disabled, pointer otherwise